content/developer/reference/cli.rst

```diff
@@ -136,13 +136,9 @@ Running the server
 
```

Comment:
Hi @nmarotte 
Why was this file changed for the upgrade documentation?

Reply:
I think some other changes got absorbed into my commit by accident. I modified this file to change the text related to the odoo shell, starting at "This shell is started with the `env` variable", as well as the first paragraph of the section.

I don't remember the rest of the changes for that file so I think they are probably from another commit @augustoPerez

Reply:
@nmarotte I adapted to keep just your changes, could you check if what I did is correct and nothing got lost?
Thanks!

Reply:
@augustoPerez perfect, looks good to me :D